bundles/org.openhab.binding.homematic/README.md

```diff
@@ -660,11 +660,18 @@ openhab:send Var_1 REFRESH
 
 In scripts:
 
-```javascript
+:::: tabs
+
+::: tab JavaScript
 import org.openhab.core.types.RefreshType
-...
-Var_1.sendCommand(RefreshType.REFRESH)
-```
+:::
+
+::: tab DSL
+
```

Comment:
Remove this line feed.

Reply:
I can't see how to do that. It seems to me as you suggested this, but how can I confirm this? Do I just hit "Resolve conversation"?

Reply:
You can either edit again and delete the empty line, or you can apply the suggested fix. After this you can mark the comment as resolved.